bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -224,12 +224,25 @@ public Collection<WorkItem> prRun(Path scratchPath) {
         // First determine if the current state of the PR has already been checked
         var seedPath = bot.seedStorage().orElse(scratchPath.resolve("seeds"));
         var hostedRepositoryPool = new HostedRepositoryPool(seedPath);
-
-        var census = CensusInstance.createCensusInstance(hostedRepositoryPool, bot.censusRepo(), bot.censusRef(), scratchPath.resolve("census"), pr,
-                                           bot.confOverrideRepository().orElse(null), bot.confOverrideName(), bot.confOverrideRef()).orElseThrow();
+        CensusInstance census = null;
         var comments = pr.comments();
         var allReviews = pr.reviews();
         var labels = new HashSet<>(pr.labelNames());
+        try {
+            census = CensusInstance.createCensusInstance(hostedRepositoryPool, bot.censusRepo(), bot.censusRef(), scratchPath.resolve("census"), pr,
+                    bot.confOverrideRepository().orElse(null), bot.confOverrideName(), bot.confOverrideRef()).orElseThrow();
+        } catch (InvalidJCheckConfException invalidJCheckConfException) {
+            if (bot.confOverrideRepository().isEmpty()) {
+                var text = " ⚠️ @" + pr.author().username() + " The `.jcheck/conf` in the target branch of this pull request is either invalid or missing completely. "
+                        + "Until that is resolved, this pull request cannot be processed. Please notify the repository owner.";
+                addErrorComment(text, comments);
+            } else {
+                var text = " ⚠️ @" + pr.author().username() + " The `.jcheck/conf` in the override repo is either invalid or missing completely. "
+                        + "Until that is resolved, this pull request cannot be processed. Please notify the repository owner.";
+                addErrorComment(text, comments);
+            }
```

Comment:
I think this is another situation which is that the override configuration is invalid or missing

Reply:
If we throw two separate exception types as suggested below, then we can provide more detailed error messages, for either a missing or an invalid jcheck conf.

In the case of the override jcheck configuration, the file can be configured to have any name. I would suggest:
"The external jcheck configuration for this repository could not be resolved. Until that is fixed, this pull request cannot be processed. Please notify a Skara admin."